Source/WebCore/page/PageOverlay.h

```diff
@@ -77,7 +79,7 @@ class PageOverlay final : public RefCounted<PageOverlay> {
 
     WEBCORE_EXPORT PageOverlayController* controller() const;
 
-    typedef uint64_t PageOverlayID;
+    using PageOverlayID = WebCore::PageOverlayID;
```

Comment:
It seems silly to keep the `PageOverlay::PageOverlayID` type now that `WebCore::PageOverlayID` exists, but that could be a separate cleanup.

Reply:
This comes up repeatedly lot in these types of patches. We can do larger patches that resolve these, but this seems a more expedient approach. I am willing to take it out later.

Reply:
Ah, got it. Thank you for clarifying.

Reply:
I’m also not sure that it’s all that helpful to have a `typedef` here that is just `uint64_t` with no type checking. We have been using type safe IDs instead in newer code.